semantic-conventions/tsconfig.esm.json

```diff
@@ -1,8 +1,14 @@
 {
   "extends": "../tsconfig.base.esm.json",
   "compilerOptions": {
+    "lib": [
+      "es2017",
+      "dom",
+      "dom.iterable"
+    ],
```

Comment:
Same question as for "semantic-conventionis/tsconfig.json": can we drop "lib" and have this still work?